streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -297,9 +298,21 @@ public void shouldNotThrowExceptionOnRestoreWhenThereIsPreExistingRocksDbFiles()
     public void shouldCallRocksDbConfigSetter() {
         MockRocksDbConfigSetter.called = false;
 
+        final Properties props = StreamsTestUtils.getStreamsConfig();
```

Comment:
This is to illustrate the usage of passing in any user-customized configurable value or shared objects to the setter. @rodesai @brianbushree